### DISCUSSION TITLE
Setting a default quality and format in configuration

### DISCUSSION THREAD

**Discussion by jonasboman** [SixLabors/ImageSharp.Web#359] [Q&A] [answered]
Hi

Could you in configurations set default format and quality and thus disable the ability to alter the settings by using querystring parameters?

**Comment by JimBobSquarePants** [comment]
You can set your own encoder via the `Configuration` type which would have default quality.

**Reply by jonasboman** [reply]
Yes, and would that also omit the querystring parameters?  
Perhaps?
c.Command.Remove("Quality")
c.Command.Remove("Format")

**Reply by JimBobSquarePants** [reply, marked as answer]
Configuration stems from ImageSharp and allows registration of custom formats. Processors are registered in your DI setup. Have a look at the sample app in the repo.
